app/controllers/api/v1/users_controller.rb

```diff
@@ -7,8 +7,12 @@ def index
   end
 
   def show
-    #verify_token(@user)
-    #render json: @user
+   token = request.env["HTTP_AUTHORIZATION"]
+    if  @user && token  && User.decode_token(token)

```

Comment:
Should we be requiring a token just to see a user's information or should this info be public?

The tests have it allowed, should we change the test or the code?